colossus/src/main/scala/colossus/protocols/http/HttpResponseParser.scala

```diff
@@ -22,24 +22,34 @@ object ResponseResult {
 object HttpResponseParser  {
   val DefaultMaxSize: DataSize = 10.MB
 
+  def static(maxResponseSize: DataSize = DefaultMaxSize): Parser[DecodedResult.Static[HttpResponse]] = maxSize(maxResponseSize, staticBody(true))
+
+  def stream(dechunkBody: Boolean): Parser[DecodedResult[StreamingHttpResponse]] = streamBody(dechunkBody)
+
+
   //TODO: eliminate duplicate code
   //TODO: Dechunk on static
```

Comment:
I'm pretty sure this comment is invalid.  I think we are already dechunking static responses.